src/components/list/list-renderer.tsx

```
@@ -63,19 +38,12 @@ export class ListRenderer {
-                this.twoLines &&
-                this.commandKey &&
-                ['small', 'x-small'].includes(this.config.iconSize),
-        };
-
         return (
-            <ul class={classNames} role={role} aria-orientation="vertical">
+            <ul
+                class="mdc-deprecated-list"
+                role={role}
+                aria-orientation="vertical"
+            >
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-orientation' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)